### ISSUE TITLE
[BUG] useTable does not return antd Form props properly

### ISSUE THREAD

**Issue by soonoo** [pankod/refine#1436]
[`searchFormProps`](https://github.com/pankod/refine/blob/master/packages/core/src/hooks/table/useTable/useTable.ts#L243-L244) from useTable hook is a [FormInstance](https://ant.design/components/form/#FormInstance), not a [Form props](https://ant.design/components/form/#Form).

```typescript
<Form
  form={..searchFormProps}
  onValuesChange={() => {
    // this does not work since searchFormProps.form dosn't exist
    searchFormProps.form?.submit(v);

    // searchFormProps.submit exists but onSearch in useTable is not triggered 
    searchFormProps.submit();
  }}
>
```

---
```typesript
const { searchFormProps } = useTable({ onSearch: () => {...} });
console.log(searchFormProps);
```
<img width="478" alt="스크린샷 2022-01-14 오후 4 42 31" src="https://user-images.githubusercontent.com/5436405/149469942-edbbb318-b643-4f62-938d-41a1dc462a3b.png">

**Comment by omeraplak** [comment]
Hey @soonoo ,
Good point! Thank you!
I've created PR #1437. I hope enough reviews come today and I can release!

**Comment by omeraplak** [comment]
Hey @soonoo  ,
We've released 2.6.2. Could you approve this fix?

**Comment by soonoo** [comment]
Hi @omeraplak 
It's good! Thanks a lot.
